napari/_qt/qt_main_window.py

```diff
@@ -49,7 +50,10 @@ def __init__(self, qt_viewer, *, show=True):
         self._qt_window.setCentralWidget(self._qt_center)
         self._qt_window.setWindowTitle(self.qt_viewer.viewer.title)
         self._qt_center.setLayout(QHBoxLayout())
-        self._status_bar = self._qt_window.statusBar()
+        # self._status_bar = self._qt_window.statusBar()
+        # if you guys approve the change i will clean this line
+        self._status_bar = QStatusBar()
+        self._qt_window.setStatusBar(self._status_bar)
```

Comment:
these changes not necessary after adding `sleep()` call to test but wanted to show basically what is needed to happen if we want to save all QWidget hierarchy from runtime gc.

Reply:
this change looks fine to me

Reply:
@sofroniewn we can merge this little change with this PR but if we want to keep alive all Qt objects from a single object, we need to do these kind of changes all over in qt folder. Would be a huge refactor.

Reply:
ah i misunderstood, ok will need to think more then if needed. thoughts @tlambert03 @jni?